[2025-04-02T14:50:30.668+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-02T14:50:30.709+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: LocalIngestionDag.ingest_dataset manual__2025-04-02T14:50:23.057568+00:00 [queued]>
[2025-04-02T14:50:30.733+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: LocalIngestionDag.ingest_dataset manual__2025-04-02T14:50:23.057568+00:00 [queued]>
[2025-04-02T14:50:30.735+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-02T14:50:30.759+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): ingest_dataset> on 2025-04-02 14:50:23.057568+00:00
[2025-04-02T14:50:30.770+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=909) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-02T14:50:30.772+0000] {standard_task_runner.py:72} INFO - Started process 916 to run task
[2025-04-02T14:50:30.776+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDag', 'ingest_dataset', 'manual__2025-04-02T14:50:23.057568+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpsq1cmove']
[2025-04-02T14:50:30.781+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask ingest_dataset
[2025-04-02T14:50:30.871+0000] {task_command.py:467} INFO - Running <TaskInstance: LocalIngestionDag.ingest_dataset manual__2025-04-02T14:50:23.057568+00:00 [running]> on host 6baa03efc864
[2025-04-02T14:50:31.064+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='LocalIngestionDag' AIRFLOW_CTX_TASK_ID='ingest_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-04-02T14:50:23.057568+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-02T14:50:23.057568+00:00'
[2025-04-02T14:50:31.068+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-02T14:50:31.092+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-02T14:50:31.094+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Hekko, world"']
[2025-04-02T14:50:31.111+0000] {subprocess.py:99} INFO - Output:
[2025-04-02T14:50:31.118+0000] {subprocess.py:106} INFO - Hekko, world
[2025-04-02T14:50:31.119+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-02T14:50:31.161+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-02T14:50:31.163+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDag, task_id=ingest_dataset, run_id=manual__2025-04-02T14:50:23.057568+00:00, execution_date=20250402T145023, start_date=20250402T145030, end_date=20250402T145031
[2025-04-02T14:50:31.247+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-02T14:50:31.293+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-02T14:50:31.297+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
